cs/cli/vowpalwabbit.cpp

```diff
@@ -79,17 +79,17 @@ VowpalWabbitPerformanceStatistics^ VowpalWabbit::PerformanceStatistics::get()
 { // see parse_args.cc:finish(...)
   auto stats = gcnew VowpalWabbitPerformanceStatistics();
 
-  if (m_vw->current_pass == 0)
+  if (m_vw->pc.current_pass == 0)

```

Comment:
pc is pretty ambiguous to me

Reply:
went for short names for now to avoid a longer name being sprinkled all over the place